Packs/SophosCentral/Integrations/SophosCentral/SophosCentral_test.py

```diff
@@ -0,0 +1,992 @@
+import json
+from datetime import datetime
+from pytest import raises
+from CommonServerPython import DemistoException
+
+BASE_URL = 'https://api-eu02.central.sophos.com'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+
+def load_mock_response(file_name: str) -> dict:
+    """
+    Load one of the mock responses to be used for assertion.
+
+    Args:
+        file_name (str): Name of the mock response JSON file to return.
+    """
+    with open(f'test_data/{file_name}', mode='r', encoding='utf-8') as json_file:
+        return json.loads(json_file.read())
+
+
+def test_sophos_central_alert_list_command(requests_mock) -> None:

```

Comment:
let's add 1 more test case where the limit is smaller than 17, and you get the expected num of alerts.

Reply:
I'm not sure what exactly it will test since the number of results returned is resolved in the API and not the integration.

Reply:
I see, resolved then.